src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "./reset.scss";
+@import "./variables.scss";
+
 body {
-  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar__wrap {
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  max-width: 725px;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  @for $day from 28 through 31 {
+    &--#{$day} &__day:nth-child(n + #{$day + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $value in $days {
+    &--#{$day} :first-child {
+      margin-left: $value;
+    }
+  }
+}
+
+.calendar__day {
+  cursor: pointer;
+  width: 100px;
+  height: 100px;
+  background-color: $color-back;
+  border: 1px solid #000;
+  transition: all 0.3s ease 0.2s;
```

Comment:
let's not use transition all, but indicate which property you use transition for